assessments/intake-process.md

```diff
@@ -0,0 +1,34 @@
+# Security Assessment Priorities & Pipeline Intake Process
+
+SIG-Security has a volunteer team of subject matter experts and industry professionals dedicated to helping SIG-Security members, the TOC, and the larger CNCF community maintain an understanding of the current state of security in the cloud native ecosystem and helping cloud native projects succeed. 
+
```

Comment:
Some formatting inconsistencies - line breaks versus line wraps.  Refer to [contributing.md](https://github.com/cncf/sig-security/blob/master/CONTRIBUTING.md)